src/index.html

```diff
@@ -4,14 +4,172 @@
+          >
+        </label>
+
+        <label class="fieldElem">
+          How old are You?
+          <input
+            type="number"
+            min="0"
+            max="100"
+            value="12"
+            id="age"
+          >
+        </label>
+
+        <label class="fieldElem">
+          Full date of birth:
+          <input
+            type="date"
+            id="dateBirth"
+          >
+        </label>
+
+        <label class="fieldElem lastElem">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            id="termsAgreed"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="fieldElem">E-mail:
+          <input
+            type="email"
+            required
+            placeholder="email@example.com"
+            id="email"
+          >
+        </label>
+
+        <label class="fieldElem  lastElem">
+          Password:
+          <input
+            type="password"
+            id="password"
+            minlength="5"
+            maxlength="16"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="fieldElem">
+          Do you love cats?
+          <label >
+            <input type="radio" name="loveCats">
+            Yes
+          </label>
+
+          <label>
+            <input type="radio" name="loveCats">
+            No
+          </label>
+        </div>
+
+        <label class="fieldElem">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </label>
+
+        <label class="fieldElem">
+          What time do you go to bed?
+          <input
+            type="time"
+            id="bedTime"
+          >
+        </label>
+
+        <label class="fieldElem">
+          What are your favorite brand of cars?
+          <select multiple id="favCars">
+            <option>BMW</option>
+            <option>Audi</option>
+            <option>Lada</option>
+          </select>
+        </label>
+
+        <label class="fieldElem lastElem">
+          How do you rate our work?
+          <input
+            type="range"
+            value="1"
+            name="workRate"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label class="fieldElem">
+          Comments:
+          <textarea
+            cols="20"
+            rows="2"
+            id="commentAddInfo">
+          </textarea>
+        </label>
+
+        <label class="fieldElem lastElem">
+          Would you recommend us?
+          <select name="recommend">
+            <option name="recommendYes" selected>yes</option>
+            <option name="recommendNo">no</option>
```

Comment:
name attr would do nothing for options, you should use `value` instead
```suggestion
            <option value="yes" selected>yes</option>
            <option value="no">no</option>
```

Reply:
VScode suggested name, so I  accepted automatically)
now everything is changed, thanks